doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md

```diff
@@ -116,6 +116,22 @@ Please observe the concatenation of entity ID and type is already given in the `
 
 [Top](#top)
 
+#### <a name="section1.2.4"></a>Summary datamodels PostgreSQL data structure
+
+This table contains the database structure used for each datamodel supported by this sink
+
+|data_model|database name|schema name|table name|
+|:----|:----|:----|:----|
+|dm-by-fixed-entity-type-database-schema|fiware-service|fiware-servicePath|entity type|
+|dm-by-fixed-entity-type-database|fiware-service|fiware-service|entity type|
+|dm-by-fixed-entity-type|agent properties|fiware-service|entity type|
+|dm-by-entity-database-schema|fiware-service|fiware-servicePath|fiware-servicePath + entity type + entity id|
+|dm-by-entity-database|fiware-service|fiware-service|fiware-servicePath + entity type + entity id|
+|dm-by-entity-type|agent properties|fiware-service|fiware-servicePath + entity type|
+|dm-by-service-path|agent properties|fiware-service|fiware-servicePath|
+|dm-by-entity|agent properties|fiware-service|fiware-servicePath + entity type + entity id|
+
```

Comment:
```suggestion

In the above table `+` means concatenation using the `_` character.
```

Moreover, I think "agent properties" should be explained a bit more. I guess it comes from "some setting in agent.conf Cygnus configuration file" but it should be detailed a little bit more.

Reply:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/2196/commits/df6b3242b05629932240c1345b6a11ee0ff6d884